addons/mass_mailing/migrations/13.0.2.0/post-migration.py

```diff
@@ -8,7 +8,20 @@
 ]
 
 
+def fill_several_campaign_id(env):
+    tables = ["mailing_mailing", "mailing_trace", "link_tracker", "link_tracker_click"]
```

Comment:
And aren't the rest of the models (except `mailing.trace`) having already filled `campaign_id`?

Reply:
It may be already filled, or it may not be filled, the cases vary. That's way I am using "`AND tt.campaign_id IS NULL`". I mean, they having old field mass_mailing_campaign_id filled doesn't guarantee they had also filled old field campaign_id.

Reply:
OK, I buy it :)